docs/mindsdb_sql/sql/create/trigger.mdx

```diff
@@ -107,6 +107,8 @@ ON mindsdb_slack.messages
         FROM chatbot_model m
         JOIN TABLE_DELTA s
         WHERE  s.user != 'U07J30KPAUF'
```

Comment:
Shall we update the ID to the placeholder, like this?
```
WHERE s.user != 'slack-bot-id' # this is to prevent the bot from replying to its own messages
```
Because usually we want to prevent the bot from replying to its own messages, that is, to exclude messages sent by the bot.

Reply:
Done!

Reply:
thanks!